sympy/__init__.py

```diff
@@ -22,6 +22,21 @@
 
 from sympy.release import __version__
 
+if 'dev' in __version__:
+    def enable_warnings():
+        import os
+        import warnings
+        action ='default'
+        travis = (os.environ.get('TRAVIS','') == 'true')
+        if travis:
+            action = 'error'
+        warnings.filterwarnings(action,   '.*',                      DeprecationWarning, module='sympy.*')
+        # let's not fail the test on that yet. 
```

Comment:
Opened. https://github.com/sympy/sympy/issues/11255

I was first trying to get the test-suite to pass.